src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  movies.map(movie => <MovieCard key={movie.imdbId} {...movie} />)
```

Comment:
давай краще писати
```suggestion
  movies.map(movie => <MovieCard key={movie.imdbId} movie={movie} />)
```
В більших завданнях, так буде зручніше розуміти що передається